charts/download-controller/values.yaml

```diff
@@ -1,4 +1,34 @@
-podAnnotations: {}
+podAnnotations:
```

Comment:
Usually we don't include the specifics here in the charts values.yaml, right?

Reply:
That is what happens when you spend hours with regex and go templates, thanks!